models/repo_mirror.go

```diff
@@ -76,17 +76,23 @@ func (m *Mirror) ScheduleNextUpdate() {
 	m.NextUpdate = time.Now().Add(m.Interval)
 }
 
+func remoteAddress(repoPath string) (string, error) {
```

Comment:
Esp. since the config-file is _not_ INI-format... If we wanna do this in pure-go we should consider using this: https://github.com/tcnksm/go-gitconfig

Reply:
I agree that parsing `.git/config` with a INI-parser is not ideal, but let's address it in a separate PR, so as to not block this backport.